src/Microsoft.ApplicationInsights.AspNetCore/ExceptionTrackingMiddleware.cs

```diff
@@ -1,42 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.AspNetCore
-{
-    using System;
-    using System.Threading.Tasks;
-    using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation;
-    using Microsoft.AspNetCore.Http;
-
-    /// <summary>
-    /// Sends telemetry about exceptions thrown by the application to the Microsoft Application Insights service.
-    /// </summary>
-    public sealed class ExceptionTrackingMiddleware
```

Comment:
Exception gets logged into `Microsoft.AspNetCore.Hosting.UnhandledException` diagnostic source and received by https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/281/files/b650efa55b39cb04fb32c6874812a6f0ba3d08ed#diff-127c7a3a6227c377b206b4904744707cR82 diagnostic listener.